drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -2007,3 +2007,21 @@ int bnxt_hwrm_pf_evb_mode(struct bnxt *bp)
 
 	return rc;
 }
+
+int bnxt_hwrm_func_bw_cfg(struct bnxt *bp, uint16_t vf,
+			uint16_t max_bw, __rte_unused uint16_t min_bw, uint16_t enables)
+{
```

Comment:
That's a bit forward looking anyway.
My plan was to add min-bandwidth to the existing API.